domains/codewithabdullah.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
```

```suggestion
  "owner": {
    "username": "mdabdullah63",
```

you need to add your github username here